src/applications/mhv/secure-messaging/components/shared/SmBreadcrumbs.jsx

```diff
@@ -102,8 +104,38 @@ const SmBreadcrumbs = () => {
                 arr.push(child);
               }
             }
+          } else if (locationBasePath === 'folder') {

```

Comment:
this is a bit of a drive-by comment so feel free to ignore if it doesn't make sense, I think this might be easier to read with a structure something like (pseudocode):

```
switch (locationBasePath) {
  case 'search':
    breadCrumbPaths.push(theData);
    break;
  default:
     breadCrumbPaths.push({
                path: `/folder/${activeFolder.folderId}`,
                label: activeFolder.name,
              });
}
```

(note the renaming of `arr` to be more descriptive)